### PR TITLE
Run scenario setup for outlines just like we do for simple scenarios

### PR DESCRIPTION
**Pull request by marnen**
This fixes the bug described in #73.

### CODE REVIEW

**Comment by marnen** [comment]
Ah, I see that Travis isn't happy. Let me fix those issues.

**Comment by marnen** [comment]
There we go. Travis likes it now. :)

File: lib/white_bread/runners/utilities.ex [hunk 1]
```diff
@@ -0,0 +1,5 @@
+defmodule WhiteBread.Runners.Utilities do
+  def apply_scenario_starting_state(feature_state, context) do
```
Comment (meadsteve) [inline lib/white_bread/runners/utilities.ex]: Nice idea. I like this being extracted 👍

Comment (marnen) [inline lib/white_bread/runners/utilities.ex]: Great! There's probably more duplication that could be reduced.

**Comment by marnen** [comment]
I resolved the merge conflicts. Possibly even correctly. :D

**Pull request merged**